accessibility.html

```diff
@@ -0,0 +1,281 @@
+              </p>
+              <p>
+                One of the first things you see when you go to the [Hoodie website] is the heading "Hoodie is for you".
+                By "you", we mean everyone. By everyone, we mean <em>everyone</em>. That brings us to this guide, which outlines how all contributors can make Hoodie accessible.
+              <p>
+            </article> 
+          </section>
+          
+          <section class="cb">
+            <article id="factors" role="article">
+              <h2>Factors affecting experiences with the web</h2>
+              <p>
+                A common misconception when we talk about accessibility is that we mean to cater for those who have long-term limited abilities or impairments <em>only</em>.
+                Often mentioned examples are blindness and motor and cognitive disabilities. It includes that, but it also includes so much more.
+              </p>
+              <p>
+                Here are some examples of things that can hinder and change your experience with the web:
+                <ul>
+                  <li>Blindness, either partial or total</li>
+                  <li>Colour blindness, of which there are many types</li>
+                  <li>Ataxia (loss of control of body movement). Each type affecting motor control in different ways</li>
+                  <li>Hearing loss, either partial or total</li>
+                  <li>Epilepsy</li>
+                  <li>Distraction</li>
+                  <li>Dyslexia</li>
+                  <li>Age</li>
+                  <li>Depression</li>
+                  <li>Tiredness</li>
+                  <li>English as a second language</li>
+                  <li>Surfing the web on a new device</li>
+                  <li>Having a bad data connection</li>
+                  <li>Having no internet connection at all</li>
+                </ul>
+              <p>
+            </article> 
+          </section>
+          
+          <section class="cb">
+            <article id="guidelines" role="article">
+              <h2>Accessibility guidelines</h2>
+              
+              <h3>Keyboard accessibility</h3>
+              <p>
+                A user should be able to navigate to all content using a keyboard. 
+                They should be able to use the TAB key to get to all inputs, links and buttons. 
+                They should be able to submit forms using the ENTER key. 
+                They should be able to open and close panels using the ENTER and ESC keys.
+              </p>
+              <p>
+                Feedback should be provided to the user when they TAB to elements, with the use of a focus states.              
+              <p>
+                
+              <h3>All content must be easy to understand</h3>
+              <p>
+                All writing should use simple language. 
```

Comment:
Do you think you need to specify how simple? (12-14yo reading level is the "default")